### DISCUSSION TITLE
readLines error

### DISCUSSION THREAD

**Discussion by 8dmartin4** [ishtms/learn-nodejs-hard-way#12] [Q&A] [answered]
Hello, I have found that when following along, during chapter 3, we use the readLines function on our file handle. When I go to run the code I get a type error stating that readLines is not a function. How do I resolve this?
![image_2023-08-22_182125871](https://github.com/ishtms/learn-nodejs-hard-way/assets/15642482/f73ca4e2-56ff-40f7-9721-5becd66ffddb)

**Comment by ishtms** [comment, marked as answer]
@8dmartin4 What Node.js version are you using? `FileHandle.readLines` was introduced in Node v18. This book is following the latest API, i.e Node v20. 

You can check the version using
```js
node --version
```

**Comment by 8dmartin4** [comment]
Ah, that would be the issue, I believe I am on version 16. Thanks for
the help!

**Reply by ishtms** [reply]
No probs 👍🏻
